src/Raven.Server/Smuggler/Documents/StreamDestination.cs

```diff
@@ -919,6 +919,11 @@ public void WriteTombstoneKey(string key)
                 Writer.WriteString(key);
                 Writer.WriteEndObject();
             }
+
+            public JsonOperationContext GetContextForNewCompareExchangeValue()
+            {
+                throw new NotSupportedException();
```

Comment:
Is it okay to throw here?

Reply:
yes, this method never called, its needed to implement the interface